examples-generated/nodebalancer/config.yaml

```diff
@@ -13,7 +13,9 @@ spec:
     checkAttempts: 3
     checkPath: /foo
     checkTimeout: 30
-    nodebalancerId: ${linode_nodebalancer.foobar.id}
```

Comment:
Not sure I understand all of these example changes. Do these make sense to you?

Reply:
the goal is to replace all of the `${linode_*}` generated examples with referenceSelector as those are situations where in most cases users will be referring between resources. this also doesn't remove the ability to use the `linode*ID` field, it just defaults the example to use the selector instead.